src/TraceEvent/TraceLog.cs

```diff
@@ -9064,11 +9241,7 @@ internal void AddILMapping(MethodILToNativeMapTraceData data)
         {
             var ilMap = new ILToNativeMap();
             ilMap.Next = ILMapIndex.Invalid;
-            var process = log.Processes.GetProcess(data.ProcessID, data.TimeStampQPC);
-            if (process == null)
-            {
-                return;
-            }
+            var process = log.Processes.GetOrCreateProcess(data.ProcessID, data.TimeStampQPC);
```

Comment:
Same here - is this a bug?  I could believe that it is.

Reply:
I can't confirm this is a bug in general, but for the realtime processing that happens with EventPipe session, it is necessary to let the process be created.